### DISCUSSION TITLE
When running latest source master, how to update to latest master?

### DISCUSSION THREAD

**Discussion by j-peeters** [beetbox/beets#4100] [Q&A] [answered]
This is a bit of a noob question. 
I recently installed the latest source version of Beets to get rid of some things that weren't working well in 1.5. 
How do I keep the source master updated when new commits have been done?

**Comment by sampsyo** [comment, marked as answer]
If you installed an "editable" version (like `pip install -e .`), then you should just need to `git pull`. Otherwise, you can reinstall with the same non-editable command you used the first time around.
